Runtime/AvatarCreator/UI/SelfieElement.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public class SelfieElement : MonoBehaviour
+    {
+        private const string TAG = nameof(SelfieElement);
+
+        [SerializeField] private ImageConfirmationElement imageConfirmationElement;
+        [SerializeField] private BodyType bodyType;
+        [SerializeField] private OutfitGender gender;
+
+        [Space(5)]
+        [Header("Events")]
+        public UnityEvent<GameObject,AvatarProperties> onAvatarCreated;
+
+        private void OnEnable()
+        {
+            imageConfirmationElement.onImageConfirmed.AddListener(OnPhotoCaptured);
```

Comment:
@HarrisonHough thoughts on this? We'd so far aimed to stay away from configuring the listeners in the code, as per you suggestion on a previous PR. Maybe we should do the same here too?

Reply:
yep good point I just mentioned similar thing in the comment below.